---
# jekyll header
---
{% include header.html %}

<!-- ============BreadCrumb=============== -->

<div id="segment-breadcrumb" class="segmnt">
  <!-- segment-breadcrumb -->
  <div class="container">
    <!-- container -->
    <ul class="breadcrumb">
      <li title="Home"><a href="index.html"><b>Home</b></a></li>
    </ul>
  </div>
  <!-- /container -->
</div>
<!-- /segment-breadcrumb -->

{% include container-start.html %}

<!-- ============CONTENT CONTENT=============== -->

<h1><a name="genomics"></a>Genomics Reporting</h1>

<p>Cancer genomics is the study of the totality of DNA sequence and gene expression differences between tumor cells and normal host cells. It aims to understand the genetic basis of tumor cell proliferation and the evolution of the cancer genome under mutation and selection by the body environment, the immune system and therapeutic interventions (source: <a href="https://www.nature.com/subjects/cancer-genomics">Nature</a>).</p>

<p>mCODE includes the minimal set of genetic related elements relevant to capture in an EHR to inform cancer assessment and treatment options. The approach is based on the <a href="https://www.hl7.org/fhir/genomics.html">HL7 CGWG Clinical Genomics Reporting Implementation Guide</a>, however, mCODE simplifies genomics reporting to only capture single discrete variants or to variants which were found in a given DNA region.</p>

<p>There are 3 profiles related to the capture of clinical genomics data:</p>
<ul>
    <li><b>GenomicsReport:</b> A diagnostic report which contain metadata about and the results for genomic analyses. Genomic reports vary in complexity and content, as simple as the results for a single discrete variant to complex sequences which are found in exome and genome-wide association studies (GWAS).</li>
    <li><b>GeneticVariantTested:</b> used to capture the results of a test for a single known variant.</li>
    <li><b>GeneticVariantFound:</b> used to record variants which could be found from tests which broadly analyze genetic regions (e.g.: exome tests) and stores results for any variants that could have been found.  If the implementer uses GeneticVariantFound, then the region in which the variant was found could be specified in the RegionStudied attribute of the GenomicsReport profile.</li>
</ul>


<h3><i>Genetic Test Identifiers</i></h3>
<p>mCODE allows for different coding systems like the Logical Observation Identifiers and Names (LOINC) and local codes to specify the genomics test or panel provided that a the coding system or ID convention is specified.  While <a href="https://www.ncbi.nlm.nih.gov/gtr/">Genetic Testing Registry (GTR)</a> is currently the best source for identifying genetic tests, the user should be aware that the GTR cannot be a sole reliable source since the test data is voluntarily updated and there is no overarching data steward. Standardization of codes for genetic tests is essential to facilitate data analysis of genetic tests, and should be a priority for the genomics testing community in the near future.</p>

<h2>Sources of Information</h2>
<p>References used in the mCODE genomics model include:</p>
<ul>
    <li><a href="https://www.hl7.org/fhir/genomics.html">HL7 FHIR Genomics Implementation Guidance</a></li>
    <li><a href="https://www.ncbi.nlm.nih.gov/gtr">The National Center for Biotechnology Genetic Test Registry (GTR)</a></li>
    <li><a href="http://www.hgvs.org/">The Human Genome Variation Society nomenclature for defining genetic variants.</li>
    <li><a href="https://www.genenames.org/">The Human Genome Organization (HUGO) Gene Nomenclature Committee</li>
</ul>

<p>&nbsp;</p>
<footer>
    <hr>
    <p><font color="#BEBEBD"><small>Approved for Public Release. Distribution Unlimited. Case Number 16-1988</small></font></p>
</footer>


<!-- ==============END CONTENT END CONTENT=================== -->

{% include container-end.html %}
{% include footer.html %}